src/test/java/hw2/Ex1.java

```diff
@@ -0,0 +1,107 @@
+package hw2;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+import java.util.List;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class Ex1 extends firstAndFinalCommonSteps {
+
+    @Test
+    public void SeleniumTestEx1() {
+
+        //2. Assert Browser title
+        assertEquals(driver.getTitle(), "Home Page");
+
+        //3. Perform login
+        driver.findElement(By.id("user-icon")).click();
+        driver.findElement(By.id("name")).sendKeys("epam");
+        driver.findElement(By.id("password")).sendKeys("1234");
+        driver.findElement(By.id("login-button")).click();
+
+        //4. Assert User name in the left-top side of screen that user is loggined
+        WebElement loginedUserName = new WebDriverWait(driver,15)
+                .until(ExpectedConditions.presenceOfElementLocated(By.id("user-name")));
+        assertEquals(loginedUserName.getText(), "PITER CHAILOVSKII");
+
+        //5. Assert Browser title
+        assertEquals(driver.getTitle(), "Home Page");
+
+        //6. Assert that there are 4 items on the header section are displayed and they have proper texts
+        WebElement homeBtn = driver.findElement(By.linkText("Home"));
+        WebElement contactFormBtn = driver.findElement(By.linkText("Contact form"));
+        WebElement serviceBtn = driver.findElement(By.linkText("Service"));
+        WebElement metalsAndColorsBtn = driver.findElement(By.linkText("Metals & Colors"));
+        assertTrue(homeBtn.isDisplayed());
+        assertTrue(contactFormBtn.isDisplayed());
+        assertTrue(serviceBtn.isDisplayed());
+        assertTrue(metalsAndColorsBtn.isDisplayed());
+        assertEquals(homeBtn.getText(), "Home");
+        assertEquals(contactFormBtn.getText(), "Contact form");
+        assertEquals(serviceBtn.getText(), "Service");
+        assertEquals(metalsAndColorsBtn.getText(), "Metals & Colors");
```

Comment:
I suppose it could be simplified. We could have some constants with the names of the menu items.